src/phpDocumentor/ConfigurationFactory/PhpDocumentor2.php

```diff
@@ -0,0 +1,181 @@
+<?php
+/**
+ * This file is part of phpDocumentor.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @copyright 2010-2015 Mike van Riel<mike@phpdoc.org>
+ * @license   http://www.opensource.org/licenses/mit-license.php MIT
+ * @link      http://phpdoc.org
+ */
+
+namespace phpDocumentor\ConfigurationFactory;
+
+use phpDocumentor\Dsn;
+
+/**
+ * phpDocumentor2 strategy for converting the configuration xml to an array.
+ */
+final class PhpDocumentor2 implements Strategy
+{
+    /**
+     * Converts the phpDocumentor2 configuration xml to an array.
+     *
+     * @param \SimpleXMLElement $phpDocumentor
+     *
+     * @return array
+     */
+    public function convert(\SimpleXMLElement $phpDocumentor)
+    {
+        $this->validate($phpDocumentor);
+
+        $extensions         = [];
```

Comment:
the default for extensions is php, php3 and phtml; by omitting content here the parser won't pick up on any file